### PR TITLE
Added 6 new icons and linked 2 apps to existing ones

### PR DESCRIPTION
**Pull request by Fxsch**
## Description

Added power icon (I hope it's okay that I gave it a universal name since it's not an app specific icon) and linked Classic Power Menu to it. Added icons for CPU Info, LocalMonero, DeutschlandCard, Easer (also linked it to Easer (beta)) and Fallout Shelter. Linked Clock You to clock and Geometry Dash Lite to geometry_dash.

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
✅ : Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories, such as copyediting)

### Icons added
* Classic Power Menu (`com.kieronquinn.app.classicpowermenu`)
* CPU Info (`com.kgurgul.cpuinfo`)
* LocalMonero (`co.localmonero.app`)
* DeutschlandCard (`de.deutschlandcard.app`)
* Easer (`ryey.easer`)
* Fallout Shelter (`com.bethsoft.falloutshelter`)

### Icons linked
* Clock You (linked `com.bnyro.clock` to `@drawable/clock`)
* Geometry Dash Lite (linked `com.robtopx.geometryjumplite` to `@drawable/geometry_dash`)
* Easer (beta) (linked `ryey.easer.beta` to `@drawable/easer`)

### CODE REVIEW

**Comment by Fxsch** [comment]
I only tested this out after opening the pull request and I've noticed that clips don't seem to work, so I'll need to fix that before this can be merged

**Comment by Fxsch** [comment]
I've fixed the issue, all icons how they're supposed to now

File: svgs/cpu_info.svg
Comment (Chefski) [inline svgs/cpu_info.svg:1]: Icons cannot use a fill design. Please change the square in the centre

**Review by Chefski — CHANGES_REQUESTED** [review]

**Comment by Fxsch** [comment]
Changed it

File: svgs/cpu_info.svg
Comment (Fxsch) [reply svgs/cpu_info.svg:1]: Changed it

**Review by Chefski — APPROVED** [review]

**Comment by Chefski** [comment]
Mind just resolving conflicts and I'll merge it then

**Comment by KTibow** [comment]
cc @Fxsch

**Comment by Fxsch** [comment]
Conflicts should be resolved

**Comment by Fxsch** [comment]
Actually, never mind. Now the `appfilter.xml` has duplicates and 2 closing tags for `resources`.

**Comment by Fxsch** [comment]
I just replaced `appfilter.xml` with the newest version from this repo and added my changes back in, I hope everything's fixed now

**Comment by Fxsch** [comment]
Would be nice if this could get merged soon, I'm already working on a new pull request

**Pull request merged**